src/lua/api-gateway/util/OauthClient.lua

```diff
@@ -16,22 +16,53 @@ function OauthClient:new(o)
     return o
 end
 
+local dogstats = require "api-gateway.dogstatsd.Dogstatsd"
+local dogstatsInstance = dogstats:new()
+
+--- metrics for Dogstatsd
+local httpCalls = 'oauth.http_calls'
+
+--- Increments the number of calls to the Oauth provider
+--  @param metric - metric to be identified in the Dogstatsd dashboard
+-- @return - void method
+--
+function OauthClient:increment(metric)
+    dogstatsInstance:increment(metric)
+end
+
+--- Measures the number of milliseconds elapsed
+-- @param metric - metric to be identified in the Dogstatsd dashboard
+-- @param ms - the time it took a call to finish in milliseconds
+-- @return - void method
+--
+function OauthClient:time(metric, ms)
+    dogstatsInstance:time(metric, ms)
+end
+
+
 function OauthClient:makeValidateTokenCall(internalPath, oauth_host, oauth_token)
     oauth_host = oauth_host or ngx.var.oauth_host
     oauth_token = oauth_token or ngx.var.authtoken
 
     ngx.log(ngx.INFO, "validateToken request to host=", oauth_host)
 
-
+    local startTime = os.time()
     local res = ngx.location.capture(internalPath, {
         share_all_vars = true,
         args = { authtoken = oauth_token }
     })
+    local endTime = os.time()
+    self:increment(httpCalls)
+
+    local elapsedTime = os.difftime(endTime,startTime) * 1000
+    local elapsedTimeMetric = 'oauth.makeValidateTokenCall.duration'
```

Comment:
there should be a standard for naming the metrics. **elapsedTimeMetric** starts with oauth.{method_name} but the metric for response status starts with oauth.http_calls.{method_name}.
This comment also applies to makeProfileCall.